spec/unit/utils/encryptable_transfer_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EncryptableTransfer do
+
+  let(:alice) { users(:alice) }
+  let(:bob) { users(:bob) }
+  let(:api_tux) { users(:api_tux) }
+  let(:bobs_private_key) { bob.decrypt_private_key('password') }
+  let(:team1) { teams(:team1) }
+
+  let(:encryptable_transfer) { described_class.new }
+
+  let(:encryptable_file) do
+    Encryptable::File.new(cleartext_file: 'fff', name: 'info.txt', content_type: 'text/plain')
+  end
+
+  context '#transfer' do
+    it 'transfers encryptable file to other user' do
+      receiver = bob
+      sender = alice
+
+      encryptable_transfer.transfer(encryptable_file, receiver, sender)
+
+      received_file = bob.personal_team.folders.find_by(name: 'inbox').encryptables.first
+      expect(received_file.encrypted_transfer_password).to be_present
+
+      personal_team_password = bob.personal_team.decrypt_team_password(bob, bobs_private_key)
+
+      encryptable_transfer.receive(received_file, bobs_private_key, personal_team_password)
+
+      expect(received_file.encrypted_transfer_password).to be_nil
```

Comment:
maybe check received_file.transferred? false ?

Reply:
It's not possible, because file.transferred? checks for encrypted_transfer_password and in this state, the transfer_password is already set to nil